Formula/goad.rb

```diff
@@ -0,0 +1,24 @@
+class Goad < Formula
+  desc "AWS Lambda powered, highly distributed, load testing tool built in Go."
+  homepage "https://goad.io/"
+  url "https://github.com/goadapp/goad.git"
```

Comment:
We need a tag here. You can should be able to use the tarball at https://github.com/goadapp/goad/archive/v1.4.0.tar.gz and remove the `version` below; Homebrew will guess it from the URL.